### DISCUSSION TITLE
Welcome to RSL-Helper Discussions!

### DISCUSSION THREAD

**Discussion by Dwza** [FarbstoffRSL/RSL-Helper#349] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kgatl1974** [comment]
Are you preparing a new build for the latest rsl patch?

**Reply by Dwza** [reply]
everytime

**Reply by Gap-Dragon** [reply]
I am getting an out of date error from rsl helper after the Raid update that dropped today.

**Comment by tplatania** [comment]
Looking for a little help getting Release 10 build 94 installed. I saw the zip files for the source codes, and I have already installed Release 9, but not understanding how to update with the source code? Is this only for the programming side and I just need to wait for the EXE for release 10? 
Thank you for help.

**Reply by Dwza** [reply]
there is a updater in the same directory

**Comment by jameselmer1** [comment]
Would be nice for you to post the actual source code of the tool in this repo.  Not sure how comfortable I am installing.  Virus Total has a malicious tag on the download url: [https://www.virustotal.com/gui/url/9b18bcac97ac8b6c9e9ababc7026e26fece898dd2197811f7a33b6a07cd1d86e/detection](https://www.virustotal.com/gui/url/9b18bcac97ac8b6c9e9ababc7026e26fece898dd2197811f7a33b6a07cd1d86e/detection)

**Reply by Dwza** [reply]
wont happen, was discussed several times :)

**Comment by draytalon** [comment]
No RSL Helper or Updater. My AV deleted the .exe files (even though i have it not to scan the folder) as it says it contained a trojan. Been using RSL for sometime (great help) so i came back on here to download as this is where i got the latest one v7 b4. Now v7 b6 is out, but all i can find is the source code, no download links. Cannot do the updater as that .exe has also been deleted. Where can i find these links so i can download again. TIA

**Reply by Dwza** [reply]
actually you can download under releases... must ne be the latest one... check recent releases untill you can download it, tell your tool to not delete the exe. Depending on what tool you are using, it my could be difficult. sometime you have to set the folder for not checking. if exe file is downloaded properly install it and than run the updater as admin and than you should be good to go.

**Comment by Niksan94** [comment]
Рус: Доброго времени суток. В заголовке обсуждения указано что можно предлагать идеи для совершенствования вашего чудо-помощника. Итак идея: при прокачке реферальных ссылок столкнулся с проблемой, что при прокачке корма, при достижения определенного уровня аккаунта всплывает уведомление что открыта новая локация, по типу "Кузница", "Рынок", "Роковая башня" и т.д. Так вот RSL с подобными уведомлениями ничего не делает, и просто останавливается. Особенно обидно когда ставишь RSL на ночь на сжигание энергии, а данное сообщение всплывает незадолго после включения программы.

En: Good day. The title of the discussion indicates that you can offer ideas for improving your miracle assistant. So the idea: when pumping referral links, I ran into a problem that when pumping feed, when a certain account level is reached, a notification pops up that a new location is open, such as "Forge", "Market", "Fatal Tower", etc. So RSL does nothing with such notifications, and just stops. It is especially disappointing when you set RSL for the night to burn energy, and this message pops up shortly after turning on the program.

**Comment by LEOBellocZane** [comment]
hi ，Igot a problem is when I run the rsl helper for auto muti- bettle or just load game data and do nothing just keep open for mins and It will gets no response of this app, just like It got crashed, no response window will occur. The thing I can do is shut it down and without any solution even I uninstall and reinstall again and again, hope can fix this issue, whatever Thanks a lot!!

**Comment by Rimmanata** [comment]
День дорый! У меня такая проблема- при загрузке RSL Helper он не видет мой ID . как исправить эту ошибку ??? 
![Screenshot_10](https://user-images.githubusercontent.com/125485293/219088437-690ca69b-63bd-4b73-a854-e7fdb39acd9b.png)
